tests/integration/update_cluster/existing_iam/kubernetes.tf

```diff
@@ -318,6 +318,11 @@ resource "aws_internet_gateway" "existing-iam-example-com" {
 resource "aws_key_pair" "kubernetes-existing-iam-example-com-c4a6ed9aa889b9e2c39cd663eb9c7157" {
   key_name   = "kubernetes.existing-iam.example.com-c4:a6:ed:9a:a8:89:b9:e2:c3:9c:d6:63:eb:9c:71:57"
   public_key = file("${path.module}/data/aws_key_pair_kubernetes.existing-iam.example.com-c4a6ed9aa889b9e2c39cd663eb9c7157_public_key")
+  tags = {
+    "KubernetesCluster"                              = "existing-iam.example.com"
+    "Name"                                           = "existing-iam.example.com"
+    "kubernetes.io/cluster/existing-iam.example.com" = "owned"
```

Comment:
Yes, I think this should be shared, because we don't delete it (we couldn't safely delete it before); so it does behave more like "shared"

Reply:
Actually, scratch that - I'm wrong!  We _do_ delete the keypairs, which is why we have the cluster name in the prefix, so `owned` is correct.